packages/docusaurus-theme-common/src/contexts/colorMode.tsx

```diff
@@ -109,12 +134,12 @@ function useContextValue(): ContextValue {
       return undefined;
     }
     const mql = window.matchMedia('(prefers-color-scheme: dark)');
-    const onChange = ({matches}: MediaQueryListEvent) => {
+    const onChange = () => {
       if (window.matchMedia('print').matches || previousMediaIsPrint.current) {
         previousMediaIsPrint.current = window.matchMedia('print').matches;
         return;
       }
-      setColorMode(matches ? ColorModes.dark : ColorModes.light);
+      setColorMode(null);
```

Comment:
🤷‍♂️ this wouldn't be my choice but I'm ok with that

I would have used something like:

```js
const storedColorMode = ColorModeStorage.get();
if (storedColorMode !== null) {
      setColorMode(matches ? ColorModes.dark : ColorModes.light);
}
```